packages/context/src/context.ts

```diff
@@ -66,6 +66,149 @@ export class Context {
     return binding;
   }
 
+  /**
+   * Create a corresponding binding for configuration of the target bound by
+   * the given key in the context.
+   *
+   * For example, `ctx.configure('controllers.MyController').to({x: 1})` will
+   * create binding `controllers.MyController:$config` with value `{x: 1}`.
+   *
+   * @param key The key for the binding that accepts the config
+   */
+  configure(key: string): Binding {
```

Comment:
Should we add an empty string as the default `key` value? 

```ts
 configure(key: string = ''): Binding {
  // ...
}
```

Sample usage:

```ts
const config = await fs.readJson('config.json');
app.configure().to(config);
```

Reply:
Fixed